docs/immortal-empires/slaanesh.md

```diff
@@ -0,0 +1,33 @@
+# Slaanesh 
+
+## N'Kari - The Arch-Temptor
+
+* **Route I - The Great Game**
+  * Though Slaanesh is presently the least of the four great Ruinous Powers, his brothers secretly fear him, for he feeds on humanity's darkest urges, a source of power that they cannot hope to match. Let the lands of Ulthuan be submerged beneath a writhing, twitching blanket of sin.
+  * Achieve the highest corruption in a minimum of 100 regions
+  * Occupy provinces: 
+    * Caledor, Tiranoc, Nagarythe, Chrace, Cothique, Northern Yvresse, Southern Yvresse, Eataine, Ellyrion, Avelorn and Saphery
+  * **Reward**:
+    * _Slaanesh Ascendant_: Allegiance points: +25% with all factions, Tribute from vassals: +100%, Devotees: +40 per turn
+Alternatives: could also buff things like the Gift of Slaanesh itself, which currently gives +5 devotees and +3 seductive influence per turn PER character's faction, so I stayed away from it because it's harder to control balance-wise
+_Reward flavour text:_ The Dark Prince's power waxes so strongly that it can no longer be contained within the Realms of Chaos. Storms howl outwards from the polar gates, driving his servants onwards to commit countless unspeakable acts in his name.
+
+* **Route II - Paradise Lost**
+  * "Come, mortal. Gorge yourself on the blood of your beloved, slain by your own hands. Yet your lust and despair mingle into a delicious concotion. Then, and only then, will you know the sweet caress of the Dark Prince."
+  * Spread 20 Marks of Slaanesh
+  * Establish 10 Cults
+  * **Reward**:
+    * _Cult Following_: Hero action cost: -30%, Hero action success chance: +20%
+_Reward flavour text:_ Decadance leads to dalliance. Dalliance leads to debauchery. Debauchery leads to death.
+
+* **Route III - The Dark Feast**
+  * Let the arrogant fools indulge themselves in fine liquors, exotic sweetmeats and the pleasures of the flesh. Then they shall feast again, and indulge in pleasures of a different sort...
+  * Vassalise 6 factions
+  * The Arch-Temptor has been thwarted by Aenarion and his hated descendants more times than he would care to admit. The time has come to visit upon them the most exquisite of agonies. Only after they beg for death will their souls be relinquished to the Dark Prince.
+  * Destroy factions:
+    * Eatine, Order of Loremasters and Naggaroth
+  * **Reward**:
+    * _Aenarion's Blood_: Attribute: Leadership: +10 when fighting against Elves, Weapon strength: +10% for N'Kari

```

Comment:
Rewards overall feel messed around to me. Maybe bonus agains Elves should be a donuts reward, and this one should get the income from vassals reward? I also can see N'Kari getting more Weapon strength after winning The Great Game.

Reply:
Done